File: Src/Client/src/app/shared/models/time-sheet-report-user-activity.model.ts
export class TimeSheetReportUserActivityModel {
  userId: number;
  fullName: string;
  activityId: string;
  displayName: string;
  groupName: string;
  activityName: string;
  totalHour: number | null;
}

export class FilterParamUserActivity {
  uIds: number[];
  pIds: number[];
  activityIds: string[];
  startDate: Date;
  endDate: Date;

  constructor(init?: Partial<FilterParamUserActivity>) {
    Object.assign(this, init);
  }
}

File: Src/Server/Kloon.EmployeePerformance.Models/TimeSheet/TimeSheetReport/TimeSheetReportUserActivityModel.cs
using System;
using System.Collections.Generic;

namespace Kloon.EmployeePerformance.Models.TimeSheet.TimeSheetReport
{
    public class TimeSheetReportUserActivityModel
    {
        public int UserId { get; set; }
        public string FullName { get; set; }
        public Guid ActivityId { get; set; }
        public string DisplayName { get; set; }
        public string GroupName { get; set; }
        public string ActivityName
        {
            get
            {
                return GroupName + ": " + DisplayName;
            }
        }
        public double? TotalHour { get; set; }
    }
    public class FilterParamUserActivity
    {
        public List<int> UIds { get; set; }
        public List<int> PIds { get; set; }
        public List<Guid> ActivityIds { get; set; }
        public DateTime? StartDate { get; set; } = new DateTime(DateTime.Now.Year, DateTime.Now.Month, 1);
        public DateTime? EndDate { get; set; } = DateTime.Now;
    }
}
